css/global.css

```
@@ -66,3 +67,58 @@ h4 {
   font-size: 13px;
   font-weight: 400;
 }
+
+.button {
+  background-color: #072ac8;
+  border: none;
+  color: white;
+  margin-top: 30px;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  font-weight: 600;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.button:hover{
+  background-color: #43d2ff;
+  border: none;
+  color: #292e47;
+  margin-top: 30px;
+  padding: 15px 32px;
+  font-weight: 600;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
```

La mayoría del código está repetido y no es necesario. Solo hace falta sobreescribir las propiedades que necesitamos cambiar cuando el button está hover.

+ uso de variables

```suggestion
  background-color: var(--color-accent);
  color: var(--color-neutral-dark);
```